vital/io/camera_from_metadata.cxx

```diff
@@ -67,12 +69,14 @@ tags_to_vector( metadata_sptr const& md, std::vector<vital_metadata_tag> tags )
 
 /// Extract coefficient metadata to a matrix
 rpc_matrix
-tags_to_matrix( metadata_sptr const& md, std::vector<vital_metadata_tag> tags )
+tags_to_matrix( metadata_sptr const& md,
+                std::vector<vital_metadata_tag> tags )
 {
   metadata_traits md_traits;
   if (tags.size() != 4)
   {
-    throw metadata_exception("Should have 4 metadata tags for RPC coefficients");
+    throw metadata_exception(
```

Comment:
If this is a kwiver exception, you could use VITAL_THROW to add source location of the exception.

Reply:
I will switch to VITAL_THROW in this file on this branch, but I noticed that the use of VITAL_THROW is not consistent throughout KWIVER.  We should probably make another branch that switches to VITAL_THROW everywhere where it can be used.